books/free-programming-books-ta.md

```diff
@@ -26,6 +26,11 @@
 * [எளிய தமிழில் VR/AR/MR](https://freetamilebooks.com/ebooks/vr_ar_mr/) - இரா.அசோகன் (PDF)
 
 
+### Big Data
+
+* [எளிய தமிழில் Big Data](https://freetamilebooks.com/ebooks/learn-bigdata-in-tamil) - து.நித்யா (PDF)
```

Comment:
In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox.
It's usually not helpful to add a category for a single resource. Leave this in hadoop

Reply:
okay sure!

Reply:
@eshellman requested changes have been committed!